components/link_tooltip/link_tooltip.tsx

```diff
@@ -109,7 +107,7 @@ export default class LinkTooltip extends React.PureComponent<Props, State> {
         };
         return (
             <React.Fragment>
-                {ReactDOM.createPortal(
```

Comment:
Since we're now only rendering this portal when show is true, could we remove the separate `visible` class applied to the div inside this?

Reply:
Yeah, you are right. Done!